1-exercises/B-setting-values/exercise2.js

```diff
@@ -26,6 +27,9 @@ let student = {
 */
 
 // write code here
+if(student.attendence>=90&& student.examScore>60){
+ student.hasPassed=true;
+}
 
```

Comment:
good job, you should/can have else as well to show false too :)

Reply:
done